administration/console.md

```diff
@@ -15,15 +15,26 @@ The console offers the option to:
 
 ## Accessing the Console
 
-Accessing the console depends on the start mode of openHAB.
-The manually start using shell/batch script ends directly in the console.
+Accessing the console depends on how openHAB was started.
+When started manually using shell or batch script, openHAB naturally transitions directly to the console.
+When started as a service, the console can be reached via local SSH.  
```

Comment:
I agree.  Shall I break the "Accessing the Console" into two sections and give a short list helping users know what method they probably started openHAB by? I'd keep it as concise as possible. Then we could add and remove from the lists as things change and more install methods or purpose built OS's are added.

Can you clarify what you mean by, "instead link SSH?"